server/terraform/README.md

```diff
@@ -25,22 +25,54 @@ This uses the `terraform@who-terraform-admin.iam.gserviceaccount.com` service ac
 - [Terraform adding credentials](https://www.terraform.io/docs/providers/google/guides/getting_started.html#adding-credentials)
 - [Securely handling keys](https://cloud.google.com/iam/docs/understanding-service-accounts?_ga=2.87249435.-2051693357.1581897767#managing_service_account_keys)
 
-### Terraform service account
+### Terraform Service Account
 
-Only needed for terraform service account setup. The account must have these permissions:
+Service accounts are used to access the cloud projects. Development account
```

Comment:
How is this account managed? It is it something manually managed in the Cloud Console before running terraform? Or by terraform? I assume the former, but is it created in the who-terraform-admin project already?